test/suite_test.go

```diff
@@ -612,14 +632,17 @@ func (e *e2e) kubectlRunOperatorNS(args ...string) string {
 }
 
 const (
-	curlBaseCMD = "curl -ksfL --retry 5 --retry-delay 3 --show-error "
-	curlCMD     = curlBaseCMD + "-H \"Authorization: Bearer `cat /var/run/secrets/kubernetes.io/serviceaccount/token`\" "
-	metricsURL  = "https://metrics.security-profiles-operator.svc.cluster.local/"
-	curlSpodCMD = curlCMD + metricsURL + "metrics-spod"
-	curlCtrlCMD = curlCMD + metricsURL + "metrics"
+	curlBaseCMD    = "curl -ksfL --retry 5 --retry-delay 3 --show-error "
+	headerAuth     = "-H \"Authorization: Bearer `cat /var/run/secrets/kubernetes.io/serviceaccount/token`\" "
+	curlCMD        = curlBaseCMD + headerAuth
+	curlHTTPVerCMD = curlCMD + "-I -w '%{http_version}' -o/dev/null "
```

Comment:
could you change this line to `curlHTTPVerCMD = curlCMD + "-I -w '%{http_version}\n' -o/dev/null "` instead the output was expecting a new line or it will keep retrying

Reply:
Oh, I missed this comment and just reverted back. I did not think the newline was a problem, because the logs show it.
```
 time="2023-11-15T16:25:41Z" level=info msg="+ /usr/local/bin/kubectl run --pod-running-timeout=5m --rm -i --restart=Never --image=registry.fedoraproject.org/fedora-minimal:latest -n security-profiles-operator pod-slichbqqgp -- bash -c curl -ksfL --retry 5 --retry-delay 3 --show-error -H \"Authorization: Bearer `cat /var/run/secrets/kubernetes.io/serviceaccount/token`\" -I -w '%{http_version}' -o/dev/null http://10.244.2.16:6060/debug/pprof/heap"
1.1pod "pod-slichbqqgp" deleted 

Note the `1.1` before the text about pod deleted.

Reply:
Weirdly enough, the return from the command is:
```
             Error:          Not equal:
                             expected: "1.1"
                             actual  : "1.1\npod \"pod-xqcgnazhrx\" deleted"
```